connexion/utils.py

```diff
@@ -70,7 +70,11 @@ def deep_get(obj, keys):
     if not keys:
         return obj
     try:
-        return deep_get(obj[int(keys[0])], keys[1:])
+        if type(obj) == dict:
```

Comment:
you can use `isinstance(obj, dict)`

Reply:
Is it more efficient?
I'm willing to understand

Reply:
Is it more efficient?
I'm willing to understand

Reply:
https://stereochro.me/ideas/type-vs-isinstance/

Reply:
I think it would be clearer to check for the unexpected case (list) and default to the expected case (dict).

Reply:
fixed